### ISSUE TITLE
VictoryArea -- Cannot read property 'left' of underfined

### ISSUE THREAD

**Issue by vnu** [FormidableLabs/victory#517]
I am trying victory for the first time and my code is pretty straightforward and I keep running into the left of undefined error.  The error, however doesn't occur for VictoryBar or VictoryPie. 

```js
return (
            <div className="row">
                <VictoryArea/>
            </div>
        );
```
#### The console error

```js
Uncaught TypeError: Cannot read property 'left' of undefined
    at totalPadding (clip-path.js:72)
    at ClipPath.calculateAttributes (clip-path.js:76)
    at ClipPath.shouldComponentUpdate (clip-path.js:49)
    at ReactCompositeComponentWrapper.updateComponent (ReactCompositeComponent.js:464)
    at ReactCompositeComponentWrapper.wrapper [as updateComponent] (ReactPerf.js:66)
    at ReactCompositeComponentWrapper.receiveComponent (ReactCompositeComponent.js:405)
    at Object.receiveComponent (ReactReconciler.js:87)
    at Object.updateChildren (ReactChildReconciler.js:84)
    at ReactDOMComponent._reconcilerUpdateChildren (ReactMultiChild.js:216)
    at ReactDOMComponent._updateChildren (ReactMultiChild.js:351)
```

**Comment by boygirl** [comment]
@vnu can you provide a little more detail. I tried to reproduce your issue, https://jsfiddle.net/boygirl/ehkyzdew/, but I'm not seeing your error.

**Comment by vnu** [comment]
@boygirl What kind of details would help you? I was able to see the graph in the js fiddle as well. I'm not sure why this is not working with my existing code and what is different between Area / Bar / Pie that Bar and Pie works but Area doesn't :/

**Comment by boygirl** [comment]
What version of Victory are you on? Are you rendering `VictoryArea` with any props? The only thing I can think of that is different between `VictoryArea` and those other components is that `VictoryArea` renders inside a container that uses `clipPath` by default so that it can animate correctly. You can check and see whether that is the problem if you add the prop `groupComponent={<g/>}` to your `VictoryArea` component and seeing if that fixes the issue.

**Comment by sisiguo** [comment]
I am having the same error and it occurs for me in the jsfiddle link posted above as well when I open up the inspector. 
<img width="1440" alt="screen shot 2017-03-15 at 3 48 50 pm" src="https://cloud.githubusercontent.com/assets/8635054/23974153/f2ca8a70-0996-11e7-8f0f-0afe3f4ac5e9.png">

**Comment by boygirl** [comment]
@vnu looks like the issue is in `victory@0.18.1` I'll get this fixed

**Comment by boygirl** [comment]
Fixed in `victory@0.18.2`

**Comment by vnu** [comment]
Thanks @boygirl for fixing this.
